bundle/tests/pipeline_glob_paths/databricks.yml

```diff
@@ -1,13 +1,8 @@
 bundle:
   name: pipeline_glob_paths
 
-resources:
-  pipelines:
-    nyc_taxi_pipeline:
-      name: "nyc taxi loader"
-      libraries:
-        - notebook:
-            path: ./dlt/*
+include:
```

Comment:
Can we keep this test as is and introduce a new acceptance test for the `../dlt/*` case. The existing test will suffice once we fix the error.

Reply:
Yeah that sounds reasonable me.